std/datetime.d

```diff
@@ -18182,6 +18171,17 @@ assert(Interval!Date(Date(1996, 1, 2), Date(2012, 3, 1)).intersection(
         return Interval(begin, end);
     }
 
+    ///
+    unittest
+    {
+        assert(Interval!Date(Date(1996, 1, 2), Date(2012, 3, 1)).intersection(
+            Interval!Date(Date(1990, 7, 6), Date(2000, 8, 2))) ==
+        Interval!Date(Date(1996, 1 , 2), Date(2000, 8, 2)));
```

Comment:
Nitpick: the indentation here looks really ugly. What about indent the above 2 lines one more level, so that the second line isn't at the same indentation as the `assert`?